### DISCUSSION TITLE
Using an Android phone to control other devices, how to use the landscape remote screen without turning on the phone's screen rotation function?

### DISCUSSION THREAD

**Discussion by hackgsl** [rustdesk/rustdesk#9628] [General]
### Bug Description

Using an Android phone to control other devices, how to use the landscape remote screen without turning on the phone's screen rotation function?
ToDesk is supported and displays the landscape interface when recognizing the phone in landscape position (no screen rotation is required)

### How to Reproduce

Every time

### Expected Behavior

null

### Operating system(s) on local side and remote side

Android-->Other

### RustDesk Version(s) on local side and remote side

1.3.1

### Screenshots

null

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/576
